components/infobox/wikis/starcraft2/infobox_person_map_maker_custom.lua

```diff
@@ -0,0 +1,101 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:Infobox/Person/MapMaker/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local PersonSc2 = require('Module:Infobox/Person/Custom/Shared')
+local Person = require('Module:Infobox/Person')
```

Comment:
```
local PersonSc2 = require('Module:Infobox/Person/Custom')
local MapMaker = require('Module:Infobox/Person/MapMaker')
```
Just my thoughts, I believe the setup should be. (note: not just a move/rename of modules)
This is comment to start a more detailed discussion how we want multi-step inheritance to work